Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_botiboti/src/main.rs

```diff
@@ -0,0 +1,16 @@
+use std::io;
+
+fn main() {
+    println!("Enter age.");
+
+    let mut guess = String::new();
```

Comment:
Nit: Why is the variable called guess if it represents an age? I think there are more suitable names to represent what the variable holds.

Reply:
copypasta :)